CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 7.5.1
+- fix a bug with tile property
+
+## 7.5.0
```

Comment:
has 7.5.0 already been published?

Reply:
introduce bug in 7.5.0 so had to release another patch